Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -632,5 +633,148 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("AreCoordinatesRelative")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddAction("Scale",
+                _("Scale"),
+                _("Modify the scale of the specified object."),
+                _("the scale"),
+                _("Size"),
+                "res/actions/scale24.png",
+                "res/actions/scale.png")
+      .AddParameter("object", _("Object"), "Drawer")
+      .UseStandardOperatorParameters("number")
+      .MarkAsAdvanced();
+
+  obj.AddExpressionAndConditionAndAction("number",
+                "ScaleX",
+                _("Scale on X axis"),
+                _("the width's scale of an object"),
+                _("the width's scale"),
+                _("Size"),
+                "res/actions/scaleWidth24.png")
+      .AddParameter("object", _("Object"), "Drawer")
+      .UseStandardParameters("number")
+      .MarkAsAdvanced();
+
+  obj.AddExpressionAndConditionAndAction("number",
+                "ScaleY",
+                _("Scale on Y axis"),
+                _("the height's scale of an object"),
+                _("the height's scale"),
+                _("Size"),
+                "res/actions/scaleHeight24.png")
+      .AddParameter("object", _("Object"), "Drawer")
+      .UseStandardParameters("number")
+      .MarkAsAdvanced();
+
+  obj.AddAction("FlipX",
+                _("Flip the object horizontally"),
+                _("Flip the object horizontally"),
+                _("Flip horizontally _PARAM0_ : _PARAM1_"),
```

Comment:
I made the changes in Sprite too, but do we loose the translation?

Reply:
We sadly loose the translation, but Crowdin will suggest the old translation because it will see that the source string is very similar. So it's "not too bad". 
Correctness in english is more important anyway.